slither/tools/properties/properties/ercs/erc20/properties/transfer.py

```diff
@@ -155,10 +155,10 @@
         description="Transfering more tokens than the balance will revert.",
         content="""
 \t\tuint balance = this.balanceOf(msg.sender);
-\t\tif (balance == (2 ** 256 - 1))
-\t\t\treturn true;
-\t\tbool transfer_other = transfer(crytic_user, balance+1);
-\t\treturn transfer_other;""",
+\t\tif (balance == (uint(-1)))
+\t\t\trevert();
+\t\ttransfer(crytic_user, balance+1);
```

Comment:
This will not catch the ERC20 tokens that return false instead of reverting. We should return the result from  `transfer`  here